test/e2e/onboarding-import.spec.js

```diff
@@ -92,4 +93,10 @@ test('should import a wallet from an existing seed', async t => {
     // Verify that we show the loading bolt and syncing page.
     .expect(loading.loadingBolt.exists)
```

Comment:
This check should be removed from here and should only come right at the end of the test